### ISSUE TITLE
Increase Rate Limit

### ISSUE THREAD

**Issue by tang-jason** [octokit/octokit.net#1466]
Hello,

Is there an octokit doc where it explains how to increase rate limit step by step? 

> If your OAuth application needs to make unauthenticated calls with a higher rate limit, you can pass your app's client ID and secret as part of the query string.

Looked at the [developer site ](https://developer.github.com/v3/#rate-limiting)but it didn't say much about where and how you obtain the "client_id" and "client_secret". 

And how to build the client with the client_id and client_secret?

**Comment by shiftkey** [comment]
@tang-jason are you referring to this limit?

> For requests using Basic Authentication or OAuth, you can make up to 5,000 requests per hour. 

Unauthenticated requests are much lower (60 per hour), and I highly recommend using authentication for interacting with the GitHub API...

**Comment by tang-jason** [comment]
@shiftkey Yes, I think so. How do I get the 5,000 requests per hour? Sorry, totally new to this.

**Comment by shiftkey** [comment]
@tang-jason if you're only interested in doing something with your account, I'd recommend creating a [Personal Access Token]() with the permissions it needs. The [Getting Started](http://octokitnet.readthedocs.io/en/latest/getting-started) guide should then help you get started with Octokit.net.

If you want to support other users running your app, you'll need to create an OAuth application so that users delegate permissions to your app and give you a token - instead of storing their credentials. This [sample](http://octokitnet.readthedocs.io/en/latest/oauth-flow/) walks you through how to integrate this into a web application.

**Comment by tang-jason** [comment]
@shiftkey Thanks so much for the link. Looking at the OAuth setup, I am not sure what to put in the Homepage URL and Authorization callback URL since this application will/might be running in the VM and triggered by task scheduler.

**Comment by tang-jason** [comment]
@shiftkey Looks like PAT will do. I got what I needed. Thanks for the pointers and help! Very appreciated. :+1:
